.bazelrc

```diff
@@ -56,6 +56,8 @@ build --enable_runfiles
 # Keep tests tagged "exclusive" in the sandbox
 test --incompatible_exclusive_test_sandboxed
 
+build --build_tests_only
```

Comment:
vsce complains in the release build when the local @angular/language-service packages are install. This flag makes it so `yarn test` only builds targets that are needed for the tests being run.